src/__tests__/store.js

```diff
@@ -76,11 +78,11 @@ describe('Store', () => {
     );
   });
 
-  test('should use "em" as default highlighting tag', () => {
+  test('should default highlighting tags should be undefined', () => {
     const store = createStore();
 
-    expect(store.highlightPreTag).toEqual('<em>');
-    expect(store.highlightPostTag).toEqual('</em>');
+    expect(store.highlightPreTag).toEqual(undefined);
+    expect(store.highlightPostTag).toEqual(undefined);
```

Comment:
Tricky decision :/
Truth is it made stuff simpler for the URL sync.
But in the end we have the same issue with the page.

If think making this return the actual tags makes more sense.

I will probably revert this tomorrow.